cmd/edgectl/aes_install.go

```diff
@@ -269,6 +278,7 @@ func (i *Installer) CaptureKubectl(name string, args ...string) (res string, err
 		err = errors.Wrap(err, name)
 	}
 	res = string(resAsBytes)
+	fmt.Println(res)
```

Comment:
That's just for debugging I presume?

Reply:
All the `kubectl` output is for debugging. I'm adding this one just for consistency. Part of the "Manage output" task (later this week) is to be able to redirect that output separately from other output, then make smart decisions about what goes where.